outputter/writer.go

```diff
@@ -0,0 +1,87 @@
+package outputter
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+)
+
+var ErrMissingFilename = fmt.Errorf("File - Filename is required")
+var ErrMissingFileManager = fmt.Errorf("File - FileManager is required")
+var ErrMissingIOWriter = fmt.Errorf("File - IOWriter is required")
```

Comment:
Error strings should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings